packages/api/tests/endpoints/test-granules.js

```diff
@@ -1869,162 +1834,166 @@ test.serial('move a file and update its UMM-G JSON metadata', async (t) => {
-      knex,
-      getFilesExistingAtLocationMethod,
+      knex: t.context.knex,
+      getFilesExistingAtLocationMethod: filesExistingStub,
     },
   };
 
   const expressResponse = buildFakeExpressResponse();
-  await put(expressRequest, expressResponse);
-  t.true(expressResponse.boom.conflict.called);
-  t.is(expressResponse.boom.conflict.args[0][0],
-    'Cannot move granule because the following files would be overwritten at the destination location: file1. Delete the existing files or reingest the source files.');
+  await patch(expressRequest, expressResponse);
+
+  t.true(
+    expressResponse.boom.conflict.calledWithMatch(
+      'Cannot move granule because the following files would be overwritten at the destination location: file1. Delete the existing files or reingest the source files.'
+    )
+  );
 });
 
-test.serial('put() with action move returns failure if more than one granule file exists', async (t) => {
-  const getFilesExistingAtLocationMethod = () => Promise.resolve([
-    { fileName: 'file1' },
-    { fileName: 'file2' },
-    { fileName: 'file3' },
-  ]);
+test.serial(
+  'PATCH with action move returns failure if more than one granule file exists',
+  async (t) => {
+    const { collectionName, collectionVersion } = t.context;
+    const filesExistingStub = () => [{ fileName: 'file1' }];
+    const granule = t.context.fakePGGranules[0];
 
-  const { insertedPgGranules, knex } = t.context;
+    const collectionId = constructCollectionId(collectionName, collectionVersion);
 
-  const body = {
-    action: 'move',
-    destinations: [{
-      regex: '.*.hdf$',
-      bucket: 'fake-bucket',
-      filepath: 'fake-destination',
-    }],
-  };
+    const body = {
+      action: 'move',
+      destinations: [
+        {
+          regex: '.*.hdf$',
+          bucket: 'fake-bucket',
+          filepath: 'fake-destination',
+        },
+      ],
+    };
 
-  const expressRequest = {
-    params: {
-      collectionId: t.context.collectionId,
-      granuleName: insertedPgGranules[0].granule_id,
-    },
-    body,
-    testContext: {
-      knex,
-      getFilesExistingAtLocationMethod,
-    },
-  };
+    const expressRequest = {
+      params: {
+        collectionId,
+        granuleName: granule.granule_id,
+      },
+      body,
+      testContext: {
+        knex: t.context.knex,
+        getFilesExistingAtLocationMethod: filesExistingStub,
+      },
+    };
 
-  const expressResponse = buildFakeExpressResponse();
-  await put(expressRequest, expressResponse);
-  t.true(expressResponse.boom.conflict.called);
-  t.is(expressResponse.boom.conflict.args[0][0],
-    'Cannot move granule because the following files would be overwritten at the destination location: file1, file2, file3. Delete the existing files or reingest the source files.');
-});
+    const expressResponse = buildFakeExpressResponse();
 
-test.serial('create (POST) creates new granule without an execution in PostgreSQL, and Elasticsearch', async (t) => {
```

Comment:
So - all of these updates are testing one of the patch methods, but not the other.   We should chat, but I'm thinking there's a code coverage issue between patch and patchByGranuleId, assuming we don't DRY up the /api/endpoints/granules.js code.   Thoughts?

Reply:
So two things:
1. The patch vs. patchByGranuleId methods are a little DRYer now as the action handling is shared. The difference is mainly in how it gets the collection. Action and update handling are shared.
2. You're right that we need to evaluate these tests especially now that we've effectively switched from PUT to PATCH in our routing. There are some tests that correctly make PUT requests but we're just routing that to the PATCH behavior. We need to figure out what to do there. I suggest doing that work in https://bugs.earthdata.nasa.gov/browse/CUMULUS-3078# as it's going to require charts and graphs.

Reply:
Put another way, I'm fairly confident that we have test _coverage_ but our test _clarity_ is lacking. I don't think it's obvious or correct in some cases what exactly is being tested.

Reply:
I'm willing to accept that, particularly given we are going to remove the granuleId only endpoint shortly.   

Adding a note in https://bugs.earthdata.nasa.gov/browse/CUMULUS-3078 and we'll call this mitigated for the purposes of *this* PR.